src/main/java/Duke.java

```diff
@@ -1,10 +1,100 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.util.Scanner;
+
+public class Duke {
+
+    private static Scanner in = new Scanner(System.in);
+
+    private static String lineInput = "";
+    private static Task[] taskList = new Task[100];

```

Comment:
Use a variable to store the number of elements in the array. Something like `MAX_SIZE`